sqlflow/client.py

```diff
@@ -7,67 +7,73 @@
 import sqlflow.proto.sqlflow_pb2 as pb
 import sqlflow.proto.sqlflow_pb2_grpc as pb_grpc
 
-_LOGGER = logging.getLogger(__name__)
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+_LOGGER = logging.getLogger()
+
+
+class Rows:
+    def __init__(self, rows_gen):
+        rows = [r for r in rows_gen()]
```

Comment:
It should not do heavy lift in __init__ method. Suggest move it to column_names and rows